google-recaptcha.html

```diff
@@ -232,6 +251,8 @@
        * The `expiredHandler` method fires the captcha-expired event.
        */
       _expiredHandler: function () {
+        this._invalid = true;
+        this.toggleClass('invalid', this._invalid);
```

Comment:
(something like [this](https://github.com/PolymerElements/iron-form/pull/60/files#diff-bed13d2e369620d26f2c23eb471d90cdR19) and [this](https://github.com/PolymerElements/iron-form/pull/60/files#diff-bed13d2e369620d26f2c23eb471d90cdR22) is what I meant by "achieving this via a declarative binding")